docs/content/docs/connectors/datastream/pulsar.md

```diff
@@ -768,6 +768,51 @@ you to reuse the same Flink job after certain "allowed" data model changes, like
 a field in a AVRO-based Pojo class. Please note that you can specify Pulsar schema validation rules
 and define an auto schema update. For details, refer to [Pulsar Schema Evolution](https://pulsar.apache.org/docs/en/schema-evolution-compatibility/).
 
+### Python API Usage
+
+Python API supports part of the functionality of Java API.
```

Comment:
What do you mean by partial functionalities? Besides, I think we could just add PyFlink examples in all the places where there is Java examples and using tab to switch between them just we have done in other places, e.g. https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/dev/table/tableapi/